README.md

```diff
@@ -106,18 +133,4 @@ Default value: `false`
 - **timeout:**  
 The amount of seconds after a task "end event" can be considered lost.  
 The task "map" is then evicted.  
-The default value is 0, which means no timeout so no auto eviction.
-
-## Need Help?
```

Comment:
If moving this to the ```BUID.md```, can we add a link to that document in this README? like use users could find it as README is the first thing they see.